mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/AuthenticationController.java

```diff
@@ -194,8 +200,19 @@ public final ResponseEntity<ForgotUsername> forgotUsername(@RequestBody ForgotUs
     return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), status);
   }
 
+  // Package private
+
+  final void dumpEncryptedLoginHashToSession(String login) {
+    if (Strings.isNotBlank(login)) {
+      String sha256hex = Hashing.sha256()
+          .hashString(login, StandardCharsets.UTF_8)
+          .toString();
+      Session.current().sput(Scope.Session, "login", sha256hex);
```

Comment:
Do we need to name this field different so when others pull it from session they know it is hashed

Reply:
ooh, good idea. Maybe 'loginHash', or something better. Other than that, I like this idea.

Reply:
very good call out! updating

Reply:
Looks good